### ISSUE TITLE
Second live

### ISSUE THREAD

**Issue by chelikol** [efroemling/ballistica#586]
### Description
Today I got a second life at the same time

### Steps to reproduce
1. Launch BombSquad
2. Play as usual

### Machine
**Platform**: Ubuntu
**BombSquad version**: [1.7.19]

### Screenshots
![Bildschirmfoto vom 2023-04-27 23-33-15](https://user-images.githubusercontent.com/66850897/234998054-ded6f89a-526e-4e72-8684-f6758b619240.png)
![Bildschirmfoto vom 2023-04-27 23-17-00](https://user-images.githubusercontent.com/66850897/234998101-5b95228c-378c-4664-8158-6603165ae3b2.png)

### Extra
I could not control my twin, but i could kill him and get points ;-)

**Comment by IvanPragma** [comment]
You played on remote public server? What version of BombSquad does it use? Are you sure that this is a bug in BombSquad and not in the plugins of this server?

**Comment by chelikol** [comment]
I took (as usual) one of the servers with a low ping.

**Comment by Silver-Volt4** [comment]
Are you perhaps using an XInput emulator (like DS4Windows)?

**Comment by chelikol** [comment]
No, my environment is linux, bombsquad native (not with wine) and ps4-controller.

**Comment by chelikol** [comment]
Now I have a "real" clone (or AI). Same character, same name. But he lives. Is it possible to create a profile with username, different only in a last invisible byte?
![Bildschirmfoto vom 2023-06-04 22-49-07](https://github.com/efroemling/ballistica/assets/66850897/8862cfc0-fc75-4540-9d87-e8ef283bff77)

**Comment by Loup-Garou911XD** [comment]
V2 accounts limit the usage of invisible characters I think,though it was possible with v1 accounts
It's either a server plugin or some player 
Not a bug in bombsquad though, probably

**Comment by MatiasMGz** [comment]
Bruh. Just close this.
